main.js

```diff
@@ -26,136 +26,99 @@ categoryBoxWrapper.addEventListener('click', activateCategory);
 startActivityButton.addEventListener('click', submitForm);
 
 //Add functions below
+function show(element) {
```

Comment:
Let's establish an order (if we haven't already) to the function list. My recommendation. (a) order based on event listener so all categoryBoxWrapper functions are first, (b) then order based on the sequence in the function called in the event listener... so for startActivityButton... 1) subitForm, 2) createInputValidity, 3) createNewActivity, 4) and so on following the functions as they are used...

Reply:
Sounds good to me!

Reply:
I added a card for this on the project board